monkestation/code/modules/hydroponics/seeds.dm

```diff
@@ -76,6 +76,9 @@
 	for(var/g in genes)
 		var/datum/plant_gene/G = g
 		S.genes += G.Copy()
+
+	for(var/datum/plant_gene/trait/traits in S.genes)
```

Comment:
To my knowledge, this is already done in Initialize- what's up here?

Reply:
This addition fixes corpse flowers not spawning miasma when grown. I am not sure why exactly it is needed but copy() also does this and it seems to work. Perhaps the seed is initialized empty before getting its genes?